webview/src/shared/components/Icon.tsx

```diff
@@ -15,9 +15,8 @@ export const Icon: React.FC<IconProps> = ({
   ...other
 }) => {
   const I = icon
-  const fill = 'magenta' // This color is used to make sure we change it in CSS
```

Comment:
[F] I think we are ok to drop this as the VS Code buttons work out of the box but happy to be corrected if you think is the wrong way to go.